batch7crud-roll7/src/main/java/com/lftechnology/batch7crud/dao/StudentDataAccess.java

```diff
@@ -0,0 +1,98 @@
+package com.lftechnology.batch7crud.dao;
+
+import com.lftechnology.batch7crud.db.DbUtilities;
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.model.Student;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Created by leapfrog on 1/18/16.
+ */
+public class StudentDataAccess {
+    public static final Logger LOGGER = Logger.getLogger("StudentDataAccessLogger");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make the "LOGGER" logger private static final. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1312)